go.mod

```diff
@@ -1,39 +1,45 @@
 module github.com/Digital-Insight-Technologies-Ltd/gin-jwt-cognito
 
-go 1.20
+go 1.21.0
```

Comment:
After updating the Go version and dependencies, run `go mod tidy` to ensure that `go.sum` is correctly updated and there are no unused dependencies. [medium]